interface/login/login.php

```diff
@@ -349,3 +370,79 @@ function transmit_form(element) {
   </form>
 </body>
 </html>
+<?php if ($GLOBALS['google_signin_enabled']) { ?>
```

Comment:
if google-signin-client_id needs to be unempty for this to work then would also ensure this in above conditional

Reply:
fixed